src/Parser.js

```diff
@@ -759,6 +704,36 @@ export default class Parser {
         };
     }
 
+    parseArgumentGroup(
+        mode?: Mode,
+    ): AnyParseNode {
+        const outerMode = this.mode;
+        // Switch to specified mode
+        if (mode) {
+            this.switchMode(mode);
+        }
+
+        // Start a new group namespace
+        this.gullet.beginGroup();
+        const firstToken = this.fetch();
+        const expression = this.parseExpression(false, "EOF");
+        const lastToken = this.fetch();
+        // End group namespace before consuming symbol after close brace
+        this.gullet.endGroup();
+        const result = {
+            type: "ordgroup",
+            mode: this.mode,
+            loc: SourceLocation.range(firstToken, lastToken),
```

Comment:
This may be a regression, as `firstToken` is the first token after the opening brace, if it exists.